src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.targets

```diff
@@ -3,16 +3,19 @@
   <PropertyGroup>
     <MicrosoftAndroidSdkTargetsImported>true</MicrosoftAndroidSdkTargetsImported>
     <_XamarinAndroidBuildTasksAssembly>..\tools\Xamarin.Android.Build.Tasks.dll</_XamarinAndroidBuildTasksAssembly>
+    <UsingAndroidNETSdk>true</UsingAndroidNETSdk>
+    <TargetPlatformSupported Condition=" '$(TargetPlatformIdentifier)' == 'Android' ">true</TargetPlatformSupported>
+    <!-- Enable nuget package conflict resolution -->
+    <ResolveAssemblyConflicts>true</ResolveAssemblyConflicts>
+    <_AndroidResourceDesigner>Resource.designer.cs</_AndroidResourceDesigner>
   </PropertyGroup>
 
-  <!-- Default property settings, imported before Microsoft.NET.Sdk -->
+  <Import Project="Microsoft.Android.Sdk.BundledVersions.props" Condition=" Exists('Microsoft.Android.Sdk.BundledVersions.props') " />
+  <Import Project="Microsoft.Android.Sdk.SupportedPlatforms.props" />
   <Import Project="Microsoft.Android.Sdk.DefaultProperties.props" />
   <Import Project="$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.Common.Debugging.props"
       Condition="Exists('$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.Common.Debugging.props')"/>
```

Comment:
@dsplaisted Am I correct to assume that `AutoImport.props` is the only pre-project file import hook that workloads have access to? It probably makes sense for us to rename these to `.targets` files now since we no longer have an `Sdk.props` import hook, and we shouldn't be setting properties in `AutoImport.props`.

Reply:
Is there a way a workload has a chance to set properties after the `.csproj`? We were setting `$(ProduceReferenceAssembly)` to `false` by default for "application" projects, as the reference assembly is somewhat useless. It needs to be set before `Microsoft.NET.Sdk/Sdk.targets` but depends on `$(OutputType)` set to `Exe` in the `.csproj`.

Likewise, I think we should set `$(GenerateDependencyFile)` to `false`? We don't need to generate a `.deps.json` file?

Reply:
> Is there a way a workload has a chance to set properties after the `.csproj`? We were setting `$(ProduceReferenceAssembly)` to `false` by default for "application" projects, as the reference assembly is somewhat useless. It needs to be set before `Microsoft.NET.Sdk/Sdk.targets` but depends on `$(OutputType)` set to `Exe` in the `.csproj`.
> 
> Likewise, I think we should set `$(GenerateDependencyFile)` to `false`? We don't need to generate a `.deps.json` file?

I discussed this a bit with @pjcollins, we may need to add another targets hook for this scenario.

Reply:
I'm running into a similar scenario for Xamarin.iOS: we have to set the `DebuggerSupport` property before Microsoft.NET.Sdk is imported, but we need to know other properties in the user's csproj to know which value to set `DebuggerSupport` to.